server/src/routes/api/v1/ReviewsRouter.js

```diff
@@ -24,4 +24,14 @@ reviewsRouter.post('/', async (req, res) => {
   }
 })
 
+reviewsRouter.delete("/:id", async (req,res)=>{
+  try {
+    await Review.query().findById(req.params.id).delete()
+
+    return res.status(204).json({ message: "review has been deleted!" })
```

Comment:
use this message in the final return on the fetch request, rather than also creating it in the fetch request - see my comment above